applications/contents/admin.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib import admin
+
+from .models import Content
+
+
+class ContentAdmin(admin.ModelAdmin):
+    list_display = ('key', 'title', )
+    search_fields = ('key', 'title', )
```

Comment:
The trailing comma is used when there is only one field specified to indicate that is a tuple.
```suggestion
    list_display = ('key', 'title')
    search_fields = ('key', 'title')
```